internal/mage/engine.go

```diff
@@ -324,6 +325,37 @@ func (t Engine) Dev(ctx context.Context) error {
 	return nil
 }
 
+func (t Engine) TestShim(ctx context.Context) error {
```

Comment:
Could you remove that before merge, it was only for testing purpose

Reply:
Done, thanks :tada: